tools/src/main/java/org/apache/kafka/trogdor/rest/CreateMultipleTasksRequest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.trogdor.rest;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * A request to the Trogdor coordinator to create mulitple tasks.
+ */
+public class CreateMultipleTasksRequest extends Message {
+    private final List<CreateTaskRequest> tasks;
+
+    @JsonCreator
+    public CreateMultipleTasksRequest(@JsonProperty("tasks") List<CreateTaskRequest> tasks) {
+        this.tasks = tasks;
```

Comment:
We should handle the case where this is null (by treating it as empty)

Reply:
I think that we should just make the `tasks` field required. When the user doesn't provide it he'll get a 400 - it doesn't make sense to accept an empty json.
He can still send an empty list and that would return 201

Reply:
I think it's easier just to treat null as empty.  Otherwise we have to worry about testing another code path that really doesn't add any value to the user

Reply:
So, following up on this discussion: the latest change is to add `required = true`.  However, this doesn't prevent someone from setting the value to to `null`.  See https://stackoverflow.com/questions/18320731/jackson-jsonpropertyrequired-true-doesnt-throw-an-exception  So we should still check for null and use an empty map here if needed.

Also, please make this unmodifiable using `Collections.unmodifiableMap`